legend/metrics_library/metrics/platform_k8s_deployment_metrics.j2

```diff
@@ -9,7 +9,7 @@ panels:
     description: current cpu utilisation per container
     targets:
       {% for dimension in data %}
-      - metric: round(((sum(rate(container_cpu_usage_seconds_total{container!~"POD", pod=~"^{{ dimension.deployment_name }}.*"}[5m])) by (container) / sum(kube_pod_container_resource_limits{pod=~"^{{ dimension.deployment_name }}.*", resource="cpu", unit="core"}) by (container)) * 100), 0.1)
+      - metric: round(((sum(rate(container_cpu_usage_seconds_total{container!~"POD", pod=~"^{{ dimension.deployment_name }}-[a-zA-Z0-9]*-[a-zA-Z0-9]*"}[5m])) by (container) / sum(kube_pod_container_resource_limits{pod=~"^{{ dimension.deployment_name }}-[a-zA-Z0-9]*-[a-zA-Z0-9]*", resource="cpu", unit="core"}) by (container)) * 100), 0.1)
```

Comment:
Check if `\w+` can be used instead. Also, it should be `+`  instead of `*`.

Reply:
Yep, \w+ is supported,  replacing the query with `pod=~"^{{ dimension.deployment_name }}-\\w+-\\w+"`